src/Illuminate/View/Factory.php

```diff
@@ -547,6 +556,34 @@ public function startSection($section, $content = '')
 	}
 
 	/**
+	 * Cache or return content from a content section.
+	 *
+	 * @param  string  $section
+	 * @param  string  $content
+	 * @return void
+	 */
+	public function cache($key, $condition = true, \Closure $closure)
```

Comment:
They don't get a choice, its passed in automatically so it will never be anything other than a closure, unless you call the method direct and i'm not sure what the use case would be for that?